index.js

```diff
@@ -3,131 +3,58 @@
 
 /* eslint-disable no-console */
 
-const fs = require('fs');
-const exec = require('child_process').exec;
-const tempWrite = require('temp-write');
-const splitLines = require('split-lines');
 const chalk = require('chalk');
-const remark = require('remark');
-const visit = require('unist-util-visit');
-const codeFrame = require('babel-code-frame');
-
-const NOT_FOUND = 127;
-const ICONS = {
-	warning: '⚠',
-	error: '✘',
-};
-const COLORS = {
-	warning: 'yellow',
-	error: 'red',
-};
+const glob = require('glob');
+const worker = require('./worker');
 
 /* ---------------- 8< -------- 8< ---------------- */
 
-const filepath = process.argv[2];
-if (!filepath) {
+const pattern = process.argv[2];
+if (!pattern) {
```

Comment:
You'll get usage instructions. It would be possible to capture and show some message then, though.